sample-input/benchmarks/LRA/surfaces.py

```diff
@@ -0,0 +1,18 @@
+import openmoc
+
+###############################################################################
+#                 Create dictionary of common surfaces
+###############################################################################
+
+surfaces = {}
+
+# Instantiate surfaces
+surfaces['Root x-min'] = openmoc.XPlane(x=-82.5, name='Root x-min')
```

Comment:
If the only surfaces correspond to the "Root" then I would prefer removing the redundant `"Root"` from all of these keys and `Surface` names.